.github/workflows/ci.yaml

```diff
@@ -0,0 +1,87 @@
+name: CI
+
+on:
+    push:
+        branches:
+            - main
+            - '[0-9]+.[0-9]+'
+    pull_request: ~
+
+jobs:
+    cs_fix:
+        name: Run code style check
+        runs-on: "ubuntu-20.04"
+        strategy:
+            matrix:
+                php:
+                    - '8.0'
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Setup PHP Action
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php }}
+                  coverage: none
+                  extensions: 'pdo_sqlite, gd'
+                  tools: cs2pr
+
+            - uses: "ramsey/composer-install@v1"
+              with:
+                  dependency-versions: "highest"
+
+            - name: Run code style check
+              run: composer run-script check-cs -- --format=checkstyle | cs2pr
+
+    integration-tests-mysql:
+        name: MySQL integration tests
+        services:
+            mysql:
+                image: mysql:5.7
```

Comment:
for 4.0 could be changed to 8.0 probably

Reply:
Good point, I will make the change during cross-merge